guides/contact-book/loading-data-with-components.md

```diff
@@ -10,9 +10,9 @@ In the constructor let's first set `contacts` to an empty array, then using `win
 we can set the value of `contacts` back to our data array after 2000 milliseconds:
 
 ```js {data-filename="app/components/contact-list.js" data-diff="+6,+7,+8,+9,+10,+11,+12,+13,+14,+15,+16,+17,+18,-19,-20,-21,-22,-23,-24,-25,-26,-27,-28"}
-export default class ContactList extends Component {
-  constructor() {
-    super(...arguments);
+export default Component.extend({
+  init() {
+    this.super(...arguments);
 
     this.listTitle = 'My Contacts';
```

Comment:
@rtablada should these be changed to `this.set`?

Reply:
eek, probably needs to be `this._super`

Reply:
@locks for the first introduction to setting properties I used regular `=` since `init` occurs before observers are set on the template. So `=` is fine. Later in the tutorial when the students are introduced to changing observed values (via `set`) we should go back and change all `=` to `this.set`.